WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/CoinJoinDetailsViewModel.cs

```diff
@@ -43,18 +41,20 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 	{
 		base.OnNavigatedTo(isInHistory, disposables);
 
-		_updateTrigger
-			.Subscribe(_ => Update())
-			.DisposeWith(disposables);
+		_wallet.Transactions.TransactionProcessed
+							.Subscribe(_ => Update())
+							.DisposeWith(disposables);
 	}
 
 	private void Update()
 	{
-		Date = _coinJoin.Transaction.DateString;
-		CoinJoinFeeAmount = UiContext.AmountProvider.Create(_coinJoin.Transaction.OutgoingAmount);
-		Confirmations = _coinJoin.Transaction.Confirmations;
-		IsConfirmed = Confirmations > 0;
-
-		TransactionId = _coinJoin.Transaction.Id;
+		if (_wallet.Transactions.TryGetById(TransactionId, out var transaction))
```

Comment:
This method is wrong, only finds parents, and not children.

![image](https://github.com/zkSNACKs/WalletWasabi/assets/16364053/e66a2ee1-1301-4ef7-b80b-37db75f0184e)

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/11817/commits/8a4e9ca4a208ccbf73b26cd1b0bc1f7db6b26598 fixes it